test_files/jsdoc_types/jsdoc_types.js

```diff
@@ -53,7 +53,6 @@ let useNeverTyped2;
 let useNeverTypedTemplated;
 /**
  * Note: JSDoc should not reference NeverTyped because the type is blacklisted.
- * @implements {InexpressibleType}
```

Comment:
I wonder if we should emit something like:

```
if (false) {
  function supertype(/** boolean */ supertype, /** ImplementsNeverTyped */ subtype) { supertype = subtype; }
}
```

That is, assign `subtype` into something nonsensical to prevents its properties to be disambiguated? This might not work at all, but we should raise the situation as a question with the Closure team on how to handle this situation. That is, how do we stop a subtype from being disambiguated if we cannot spell out its supertype?

Reply:
The part that is scary here is that if we can't spell out its supertype, then not only do we need to prevent it from getting ambiguated, but we *also* need the supertype to not get mangled so that it matches at runtime.  I think in the presence of mapped types in particular this gets pretty gross.